unic/ucd/bidi/src/bidi_class.rs

```diff
@@ -109,7 +109,9 @@ pub enum BidiClassCategory {
 impl BidiClass {
     /// Find the character *Bidi_Class* property value.
     pub fn of(ch: char) -> BidiClass {
-        bsearch_range_value_table(ch, BIDI_CLASS_TABLE)
+        // UCD/extracted/DerivedBidiClass.txt: "All code points not explicitly listed
+        // for Bidi_Class have the value Left_To_Right (L)."
+        *BIDI_CLASS_TABLE.find_or(ch, &L)
```

Comment:
If we always pack the [complex default values](http://unicode.org/reports/tr44/#Default_Values_Table) into the tables, then definitely yes. The only reason `find_or_else` exists is an expectation that it might be used for one of these complex defaults.